.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @KittyCAD/frontend
```

Comment:
@franknoirot this is what's blocking me to merge this. The team isn't available somehow. Might be a public repo thing vs private team

Reply:
![image](https://github.com/user-attachments/assets/e2552581-8c3b-46f1-a3a7-559b347c0410)
 we back